drivers/counter/Kconfig.nrfx

```diff
@@ -112,6 +112,16 @@ config COUNTER_RTC0_PRESCALER
 	help
 	  Frequency = 32768 / (prescaler+1).
 
+config COUNTER_RTC0_PPI_WRAP
+	bool "Enable PPI wrapping in RTC0"
+	select NRFX_PPI if HAS_HW_NRF_PPI
+	select NRFX_DPPI if HAS_HW_NRF_DPPIC
+	help
+	  If enabled, overflow different than full range (24 bits) is handled
```

Comment:
Would it make sense for this to be a boolean property in the device tree instead?

Reply:
there are more candidates here to be moved to device tree. I will do all in separate PR.